package.json

```diff
@@ -93,6 +93,10 @@
   "activationEvents": [
     "*"
   ],
+  "extensionKind": [
+    "ui",
```

Comment:
hey @mikekosulin,  is there any specific reason for choosing this order and not ["workspace", "ui"]?

Reply:
Yeah, I’ve checked a few options and ‘["workspace", "ui"]‘ doesn’t affect on the issue. So Tabnine extension activates if setting is ‘["ui", "workspace"]‘ or ‘["ui"]’ 

You can easily check it by following these steps to reproduce:
1. Start any devcontainer
2. Install Tabnine extension 
3. Add the the code below to your settings and reload the VS Code window:
‘’’
// VSCode user's settings.json
{
  "remote.extensionKind": {
    "TabNine.tabnine-vscode": [
      "workspace",
      "ui"
    ]
  }
}
‘’’